frontend/src/components/shared/feedbackField.js

```diff
@@ -0,0 +1,35 @@
+import Alert from "@mui/material/Alert";
+import Box from "@mui/material/Box";
+
+export const FeedbackField = (props) => {
+  const response = props.response;
+
+  if (response == null) return;
```

Comment:
we should also provide a response when a user provides an existing username, but an invalid password